.github/workflows/ci.yaml

```diff
@@ -0,0 +1,745 @@
+name: CI
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches:
+      - dev
+      - rc
+      - master
+  pull_request: ~
+
+env:
+  DEFAULT_PYTHON: 3.7
```

Comment:
```suggestion
  DEFAULT_PYTHON: 3.7.1
```

Reply:
For linting it's important we use the lowest possible version that we allow.

Reply:
Nope, this is not used for tests, but merely the runtime for e.g., linting. It refers to the versioning for the Python interpreter provided by GitHub.

For the tests, our Azure CI containers are used, which provide the needed versions (and use 3.7 & 3.8 as their tags).